Source/Scene/Cesium3DTile.js

```diff
@@ -553,11 +558,18 @@ define([
      */
     Cesium3DTile.prototype.createBoundingVolume = function(boundingVolumeHeader, transform, result) {
         var center;
+        var hasTileTransform = !Matrix4.equals(this.computedTransform, Matrix4.IDENTITY);
         if (defined(boundingVolumeHeader.box)) {
             var box = boundingVolumeHeader.box;
             center = Cartesian3.fromElements(box[0], box[1], box[2], scratchCenter);
             var halfAxes = Matrix3.fromArray(box, 3, scratchHalfAxes);
 
+            if (hasTileTransform) {
+                // Transform from y-up to z-up
+                Matrix3.multiply(yUpToZUpMatrix3, halfAxes, halfAxes);
+                Matrix4.multiplyByPoint(yUpToZUpMatrix4, center, center);
+            }
```

Comment:
Let's do it.  Please update the spec as well - including the main spec as well as `b3dm` and `i3dm` specs where glTF is explicitly mentioned.